attendance/API_HR/c-holiday.php

```diff
@@ -0,0 +1,39 @@
+<?php 
+
+trait Holiday {
+    
+    public static function addHoliday($name, $date, $type){
+
+        $r_error = 0;
+        $r_data = array();
+        $return = array();
+
+        $q = "SELECT * from holidays where date = '$date'";
```

Comment:
add a check the if all the params are not empty then only it should process, else error message that fields are empty